beautifultable/beautifultable.py

```diff
@@ -30,6 +30,9 @@
 import copy
 import csv
 import warnings
+from numpy import r_
```

Comment:
why is this import needed? don't see it getting used.

Reply:
Removed.